tools/analyze-chain.py

```diff
@@ -118,9 +119,9 @@ def main(
 
         ref_lookup_time = time() - start_time
 
-        flags: int
+        flags = ENABLE_KECCAK
         if mempool_mode:
-            flags = MEMPOOL_MODE
+            flags |= MEMPOOL_MODE
         else:
             flags = 0
```

Comment:
Oh, looks like this is wrong, this disables Keccak as well when not using MEMPOOL_MODE.